frontend/src/components/cart-page/CartItemsSection.tsx

```diff
@@ -0,0 +1,50 @@
+import { useContext } from "react";
+import { useNavigate } from "react-router-dom";
+
+import { ShoppingCartContext } from "../../contexts/shopping-cart-context/shoppingCartContext";
+import { Card, CardContent, Typography } from "@mui/material";
+
+import cartStyles from "./cartStyles";
+import ItemsList from "../checkout/place-order/order-itmes/ItemsList";
+import EmptyCartMessage from "./EmptyCartMessage";
+
+const { cartCardStyle } = cartStyles;
+
+const CartItemsSection = () => {
+    const navigate = useNavigate();
+    const { cartItems } = useContext(ShoppingCartContext);
+
+    const navigateHome = () => navigate("/");
+
+    const renderCartContent = () => {
+        const isCartEmpty = cartItems.length === 0;
```

Comment:
` const isCartEmpty = cartItems.length === 0;`
this will throw an error if cartItems is undefined, let's use:
` const isCartEmpty = cartItems?.length === 0;`

Reply:
Resolved